openqabot/types/incident.py

```diff
@@ -35,12 +39,14 @@ def __init__(self, incident):
         self.channels += [
             Repos(p, v, "x86_64")
             for p, v in (
-                r.split(":")[2:]
-                for r in (
-                    i
-                    for i in incident["channels"]
-                    if i.startswith("SUSE:Updates") and "openSUSE-SLE" in i
+                val
+                for val in (
+                    r.split(":")[2:]
+                    for r in (
+                        i for i in incident["channels"] if i.startswith("SUSE:Updates")
+                    )
                 )
+                if len(val) == 2
```

Comment:
a prime example of a magic number I would way. This should be avoided. Maybe at least you can extract that into a function with a good name explaining this?

Reply:
This isnt magic number:) it simply length of tuple 'Val'